### DISCUSSION TITLE
支持的transformers版本可以更新到4.39以上吗w

### DISCUSSION THREAD

**Discussion by IcyFeather233** [InternLM/lmdeploy#1496] [General]
我用 LLaMA-Factory 的项目对 Qwen72bchat微调了一下，然后用LMDeploy的时候先是显示：
<img width="1368" alt="image" src="https://github.com/InternLM/lmdeploy/assets/56245435/82db2b10-85fe-4f79-ae74-d58fcc3748e0">

然后我更新了transformers到4.39.2，又显示：
<img width="1412" alt="image" src="https://github.com/InternLM/lmdeploy/assets/56245435/a9532999-918f-4759-80ee-a7da130f3e58">
然后我把LMDeploy更新到4.0.0最新版了，运行显示：
<img width="1200" alt="image" src="https://github.com/InternLM/lmdeploy/assets/56245435/aab19d96-8fbc-4bbd-b203-095a33a0c811">

**Comment by lvhan028** [comment]
用 lmdeploy v0.4.0，transformers可以升级到 4.39.0
